README.md

```diff
@@ -21,15 +21,19 @@ Identity::Hostdata.domain
 # => "login.gov"
 ```
 
-Set configs from YML files in S3
+Set configs from YML files in S3 orSSM
 
 ```ruby
 Identity::Hostdata.load_config!(
   app_root: Rails.root,
   rails_env: Rails.env
 ) do |builder|
+  # YML
   builder.add(:some_option, type: :string)
   builder.add(:other_option, type: :json)
+
+  # SSM
+  builder.add_ssm(:prop_name, 'ssm-name-goes-here', type: :string)
```

Comment:
Idea: If all else is the same,  could we do something like this and key off `ssm_name` ?

```suggestion
  builder.add(:prop_name, ssm_name: 'ssm-name-goes-here', type: :string)
```

Reply:
I like this idea! will incorporate